### PR TITLE
[Merged by Bors] - chore(RingTheory/WittVector/Basic): remove porting notes

### PR DESCRIPTION
**Pull request by joneugster**
Remove a couple uncategorised, superfluous porting notes.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/WittVector/Basic.lean [hunk 1]
```diff
@@ -158,7 +158,6 @@ private def ghostFun : 𝕎 R → ℕ → R := fun x n => aeval x.coeff (W_ ℤ
 section Tactic
 open Lean Elab Tactic
 
--- porting note: removed mathport output related to meta code.
```
Comment (joneugster) [inline Mathlib/RingTheory/WittVector/Basic.lean:161]: The mathport output was simply saying that mathport could not translate the tactic, so this is superfluous:

```
/- ./././Mathport/Syntax/Translate/Tactic/Mathlib/Core.lean:38:34: unsupported: setup_tactic_parser -/

/- ./././Mathport/Syntax/Translate/Expr.lean:330:4: warning: unsupported (TODO): `[tacs] -/
/- ./././Mathport/Syntax/Translate/Expr.lean:330:4: warning: unsupported (TODO): `[tacs] -/
/- ./././Mathport/Syntax/Translate/Expr.lean:330:4: warning: unsupported (TODO): `[tacs] -/
```

See [here](https://github.com/leanprover-community/mathlib4/pull/4688/files/04b3759e3d8c3a63e42eae8d87cee015f6a2410c..550dc19a16a7e458722ee1f70a9af63ffdabf0a8#diff-848ab07015c9501300900be871ebed810000f8470fb5ddbda9c161d4e3641dbaL171-L173).

File: Mathlib/RingTheory/WittVector/Basic.lean [hunk 2]
```diff
@@ -256,8 +251,6 @@ private def ghostEquiv' [Invertible (p : R)] : 𝕎 R ≃ (ℕ → R) where
 
 @[local instance]
 private def comm_ring_aux₁ : CommRing (𝕎 (MvPolynomial R ℚ)) :=
-  -- Porting note: no longer needed?
-  -- letI : CommRing (MvPolynomial R ℚ) := MvPolynomial.commRing
```
Comment (joneugster) [inline Mathlib/RingTheory/WittVector/Basic.lean:260]: I think this requires no action. The required instance is now called `MvPolynomial.instCommRingMvPolynomial`
and it is synthesised fine:

```
[Meta.synthInstance] ✅ CommRing (MvPolynomial R ℚ) ▼
  [] new goal CommRing (MvPolynomial R ℚ) ▶
  [] ❌ apply inst✝ to CommRing (MvPolynomial R ℚ) ▶
  [] ❌ apply inst✝¹ to CommRing (MvPolynomial R ℚ) ▶
  [] ❌ apply inst✝² to CommRing (MvPolynomial R ℚ) ▶
  [] ✅ apply @MvPolynomial.instCommRingMvPolynomial to CommRing (MvPolynomial R ℚ) ▶
  [] result MvPolynomial.instCommRingMvPolynomial 
```

File: Mathlib/RingTheory/WittVector/Basic.lean [hunk 3]
```diff
@@ -219,14 +218,10 @@ private theorem ghostFun_intCast (i : ℤ) : ghostFun (i : 𝕎 R) = i :=
 alias ghostFun_int_cast := ghostFun_intCast
 
 private lemma ghostFun_nsmul (m : ℕ) (x : WittVector p R) : ghostFun (m • x) = m • ghostFun x := by
-  -- porting note: I had to add the explicit type ascription.
-  -- This could very well be due to my poor tactic writing!
-  ghost_fun_tac m • (X 0 : MvPolynomial _ ℤ), ![x.coeff]
+  ghost_fun_tac m • (X 0), ![x.coeff]
```
Comment (joneugster) [inline Mathlib/RingTheory/WittVector/Basic.lean:221]: This (and the same below) seems to have been solved in the meantime.

**Review by eric-wieser — APPROVED** [review]
bors merge

Thanks!

**Pull request closed without merging**